viewer/templates/view.html.j2

```diff
@@ -31,6 +32,7 @@ var opinions = {{ opinions }};
 </div>
 <div class="vote">
 <H2 id="opinion-h2">Opinion for CSD #{{ csd }}</H2>
+<H3 id="opinion-count"># of existing votes: {{ opinion_count }}</H3>
```

Comment:
```suggestion
<div id="opinion-count"># of existing votes: {{ opinion_count }}</div>
```
Logically, this is not a sub-heading.  Use a `<div>` instead .  You can attach style data to the div in the CSS to make the text larger and/or bold, if that's what you're trying to do, via something like:
```
div#opinion-count {
   font-size: 1.17em;
   font-weight: bold;
}
```
(These are the `<h3>` style defaults AFAICT.)

Reply:
Or maybe a `<span>` depending on where you want it to fit.  Adding a new `<div>` may end up adding more vertical height to the UI header, which is already rather large.